assets/json/campsitesfinal.json

```diff
@@ -2957,7 +2957,7 @@
     "state": "",
     "country": "England",
     "coordinates": "50.770600, 0.281761",
-    "photoUrl": "http://imgur.com/6QE55Uk"
+    "photoUrl": "https://scontent-lht6-1.xx.fbcdn.net/v/t1.0-9/20841032_227623511095821_3343519607225282963_n.jpg?oh=498211ae598a12ce1e70452f548b624c&oe=5A86C6AA"
```

Comment:
Hey! :) Thanks for your contribution, and your patience as its taken me forever to notice these pending PRs.
Ideally, these pictures would be of the group itself (i.e. its people). Maybe you'd have a link to one of those?